eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/MessageSender.java

```diff
@@ -0,0 +1,152 @@
+package io.eventuate.local.common;
+
+import io.eventuate.cdc.producer.wrappers.DataProducer;
+import io.eventuate.common.eventuate.local.BinLogEvent;
+import io.eventuate.local.common.exception.EventuateLocalPublishingException;
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.LinkedList;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class MessageSender<EVENT extends BinLogEvent> {
```

Comment:
Where is this class used?

Reply:
It is not used anymore, I will remove it.